src/include/sof/lib/dai-legacy.h

```diff
@@ -254,7 +254,7 @@ struct dai_type_info {
 /* class (driver) level (no device object) tracing */
 
 #define dai_cl_err(drv_p, __e, ...)		\
-	trace_dev_err(trace_dai_dvr_get_tr_ctx,	\
+	trace_dev_err(trace_dai_drv_get_tr_ctx,	\
```

Comment:
Apparently unused code as this could have not compiled. Based on git history, the dai_cl_ family of macros has never been used in the codebase.

Reply:
same thinking, this is the reason I submit this patch, I will try to check and remove.
Then try to align with zephyr logging interface.